MapboxNavigation/Geometry.swift

```diff
@@ -82,12 +82,12 @@ extension CLLocationCoordinate2D {
     }
     
     /// Returns the direction from the receiver to the given coordinate.
-    func direction(to coordinate: CLLocationCoordinate2D) -> CLLocationDirection {
+    public func direction(to coordinate: CLLocationCoordinate2D) -> CLLocationDirection {
```

Comment:
In that case, can we factor out Geometry.swift as a separate target (and thus module) shared between the UI and non-UI libraries?